rts/build/cmake/TestCXXFlags.cmake

```diff
@@ -137,10 +137,15 @@ IF    (NOT MSVC AND NOT DEFINED MARCH)
 	# 64bit
 	if    ((CMAKE_SIZEOF_VOID_P EQUAL 8) AND (NOT MARCH))
 		# always syncs with 32bit
-		check_cxx_accepts_flag("-march=x86_64" HAS_X86_64_FLAG_)
+		check_cxx_accepts_flag("-march=x86-64" HAS_X86_64_FLAG_)
 		if    (HAS_X86_64_FLAG_)
-			set(MARCH "x86_64")
+			set(MARCH "x86-64")
 		endif (HAS_X86_64_FLAG_)
+		# MCST lcc compiler accept -march=elbrus-v2/v3/v4/v5/v6 and -march=native
+		check_cxx_accepts_flag("-march=elbrus-v2" HAS_E2K_FLAG_)
```

Comment:
This assumes you always want to compile to the native machine and never cross-compile.

Reply:
Yes, by default, the e2k MCST lcc compiler turn on -march with a value corresponding to the architecture of the processor on which the compiler is currently running.
The x86 MCST lcc cross-compiler turn on -march with a value corresponding to the architecture of the e2k target-processor (that was tuned by default).
Therefore, the optimal value will be "-march=native" for both cases.